src/main/kotlin/io/github/inductiveautomation/kindling/utils/Serializers.kt

```diff
@@ -29,3 +31,19 @@ object ThemeSerializer : KSerializer<Theme> {
     override fun serialize(encoder: Encoder, value: Theme) = encoder.encodeString(value.name)
     override fun deserialize(decoder: Decoder): Theme = Theme.themes.getValue(decoder.decodeString())
 }
+
+object ToolSerializer : KSerializer<Tool> {
```

Comment:
Would be nice if there were less boilerplate on these simple class -> primitive serializers, but I don't think there's any way to avoid it